ansible_collections/arista/avd/roles/eos_validate_state/anta_integration.md

```diff
@@ -1,21 +1,17 @@
 ---
 # This title is used for search results
-title: Ansible Collection Role eos_validate_state - Preview Integration with ANTA
+title: Ansible Collection Role eos_validate_state - Integration with ANTA
 ---
 <!--
   ~ Copyright (c) 2023-2024 Arista Networks, Inc.
   ~ Use of this source code is governed by the Apache License 2.0
   ~ that can be found in the LICENSE file.
   -->
 
-# eos_validate_state - Preview Integration with ANTA
+# eos_validate_state - Integration with ANTA
 
-!!! warning
-    eos_validate_state integration with ANTA is in preview. Everything is subject to change.
-    If you have any questions, please leverage the GitHub [discussions board](https://github.com/aristanetworks/avd/discussions)
-
-!!! warning
-    ANTA version has been bumped to **0.14.0**. Please make sure you are running this exact version! For more details, please see the [installation section](#how-to-run-eos_validate_state-in-anta-mode).
+!!! note
+    [ANTA v1.0.0](https://anta.arista.com/stable/) is out!
```

Comment:
```suggestion
    [ANTA](https://anta.arista.com/stable/) will be the future default framework leveraged by AVD for network testing and validation. Since it introduces small breaking changes, you have to opt-in to leverage ANTA with `eos_validate_state` by configuring: `use_anta: true`.
```

Reply:
udpated